jdaviz/configs/mosviz/tests/test_parsers.py

```diff
@@ -180,7 +181,7 @@ def test_nircam_parser(mosviz_helper, tmp_path):
     '''
 
     # Download data
-    test_data = 'https://stsci.box.com/shared/static/itk7pav073nubwn58pig002m9796qzpw.zip'
+    test_data = 'https://stsci.box.com/shared/static/g3tymg7c1yj67gy0hpfreo80inqdm1ps.zip'
```

Comment:
What is the actual file name for this one?

Reply:
Whoops, good catch.